### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by agustinaGallo97**
## :link: Jira Tracker reference:
* Update the README file - https://androidtraining.atlassian.net/browse/AT-109

## Description:
Updating README file

### CODE REVIEW

**Review by mirland — APPROVED** [review]
Maybe you can add a link with the jira link and also with the zeplin link. What do you think? @FerrariBruno @f7deleon

**Comment by FerrariBruno** [comment]
Yes I agree

File: README.md [hunk 1]
```diff
@@ -1,7 +1,28 @@
-# TaskLoans Project
+# Android Template project
```
Comment (FerrariBruno) [inline README.md]: taskloans

File: README.md [hunk 2]
```diff
@@ -1,7 +1,28 @@
...
 
-TaskLoans is a project that consists of managing the washing of dishes in a company
+This is [Xmartlabs](https://xmartlabs.com) Android Template project.
```
Comment (FerrariBruno) [inline README.md:3]: taskloans description

File: README.md [hunk 3]
```diff
@@ -1,13 +1,25 @@
...
+# Taskloans description
+
+TaskLoans is a project that consists of managing the washing of dishes in a company
+
 ## Style guide
 
 Please, remember to always use and follow our [style guide](https://github.com/xmartlabs/Android-Style-Guide)
 when making contributions to this project.
 
+## :link: Jira Tracker reference:
```
Comment (FerrariBruno) [inline README.md]:
```suggestion
## :link: Jira reference:
```

File: README.md [hunk 4]
```diff
@@ -1,13 +1,25 @@
...
+
+https://androidtraining.atlassian.net/browse/AT-107
```
Comment (FerrariBruno) [inline README.md]:
```suggestion
https://androidtraining.atlassian.net
```

File: README.md [hunk 5]
```diff
@@ -1,13 +1,25 @@
-# Android Template project
+# Taskloans project
 
 This is [Xmartlabs](https://xmartlabs.com) Android Template project.
 Please read through this README before start working.
 
+# Taskloans description
+
+TaskLoans is a project that consists of managing the washing of dishes in a company
```
Comment (FerrariBruno) [inline README.md]:
```suggestion
TaskLoans is a project that helps users in a specific company to keep track of dishwashing chores.
```

**Review by FerrariBruno — APPROVED** [review]

**Pull request merged**